src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -37,227 +114,2413 @@ const ActorInit En_Ossan_InitVars = {
+    gSPSegment(POLY_OPA_DISP++, 0x09, SEGMENTED_TO_VIRTUAL(gGoronCsMouthNeutralTex));
+    SkelAnime_DrawFlexOpa(globalCtx, this->skelAnime.skeleton, this->skelAnime.jointTable, this->skelAnime.dListCount,
+                          NULL, NULL, this);
+    EnOssan_DrawCursor(globalCtx, this, this->cursorX, this->cursorY, this->cursorZ, this->drawCursor);
+    EnOssan_DrawStickDirectionPrompts(globalCtx, this);
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_oB1.c", 4476);
+}
+
+s32 EnOssan_OverrideLimbDrawZoraShopkeeper(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                                           void* thisx) {
+    EnOssan* this = THIS;
+
+    if (limbIndex == 15) {
+        rot->x += this->headRot;
+    }
+    return 0;
+}
+
+void EnOssan_DrawZoraShopkeeper(Actor* thisx, GlobalContext* globalCtx) {
+    static u64* sZoraShopkeeperEyeTextures[] = { gZoraEyeOpenTex, gZoraEyeHalfTex, gZoraEyeClosedTex };
+    EnOssan* this = THIS;
+    s32 pad;
+
+    OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_oB1.c", 4506);
+
+    func_80093D18(globalCtx->state.gfxCtx);
+    gDPSetEnvColor(POLY_OPA_DISP++, 0, 0, 0, 255);
+    gSPSegment(POLY_OPA_DISP++, 0x0C, func_80AC801C(globalCtx->state.gfxCtx));
+    gSPSegment(POLY_OPA_DISP++, 0x08, SEGMENTED_TO_VIRTUAL(sZoraShopkeeperEyeTextures[this->eyeTextureIdx]));
+
+    SkelAnime_DrawFlexOpa(globalCtx, this->skelAnime.skeleton, this->skelAnime.jointTable, this->skelAnime.dListCount,
+                          EnOssan_OverrideLimbDrawZoraShopkeeper, NULL, this);
+    EnOssan_DrawCursor(globalCtx, this, this->cursorX, this->cursorY, this->cursorZ, this->drawCursor);
+    EnOssan_DrawStickDirectionPrompts(globalCtx, this);
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_oB1.c", 4531);
+}
+
+void EnOssan_DrawPotionShopkeeper(Actor* thisx, GlobalContext* globalCtx) {
+    static u64* sPotionShopkeeperEyeTextures[] = { gPotionShopkeeperEyeOpenTex, gPotionShopkeeperEyeHalfTex,
+                                                   gPotionShopkeeperEyeClosedTex };
+    EnOssan* this = THIS;
+    s32 pad;
+
+    OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_oB1.c", 4544);
+
+    func_80093D18(globalCtx->state.gfxCtx);
+    gSPSegment(POLY_OPA_DISP++, 0x08, SEGMENTED_TO_VIRTUAL(sPotionShopkeeperEyeTextures[this->eyeTextureIdx]));
+    SkelAnime_DrawFlexOpa(globalCtx, this->skelAnime.skeleton, this->skelAnime.jointTable, this->skelAnime.dListCount,
+                          NULL, NULL, this);
+    EnOssan_DrawCursor(globalCtx, this, this->cursorX, this->cursorY, this->cursorZ, this->drawCursor);
+    EnOssan_DrawStickDirectionPrompts(globalCtx, this);
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_oB1.c", 4564);
+}
+
+void EnOssan_DrawHappyMaskShopkeeper(Actor* thisx, GlobalContext* globalCtx) {
+    static u64* sHappyMaskShopkeeperEyeTextures[] = { gOsEyeClosedTex, gOsEyeOpenTex };
```

Comment:
```suggestion
    static void* sHappyMaskShopkeeperEyeTextures[] = { gOsEyeClosedTex, gOsEyeOpenTex };
```
Im sure you get the picture by now so this will be the last one I do.